packages/flutter/test/painting/decoration_test.dart

```diff
@@ -126,6 +126,17 @@ void main() {
     expect(c.color, equals(b.color));
   });
 
+  test('Decoration equality', () {
+    // ignore: prefer_const_constructors, prefer_const_literals_to_create_immutables
+    final BoxDecoration a = BoxDecoration(color: Color(0xFFFFFFFF), boxShadow: <BoxShadow>[BoxShadow()]);
```

Comment:
I forced non-const constructors here because I'm not sure if `operator ==` is called on equal const objects. Am I wrong?